src/components/accessories/billsRecap/BillsRecap.tsx

```diff
@@ -323,70 +327,66 @@ export const BillsRecap: FC = () => {
           />
         </div>
         <div className="bills__recap__items">
-          {dataByYear.length > 0 ? (
-            [
-              <div className="bills__recap__item inline">
-                <Bar
-                  options={getOptions(
-                    t("bill.bestsellingbyquantity"),
-                    true,
-                    isMobile(width) ? 1 : 3,
-                    year.value
-                  )}
-                  data={getDataFromObject(
-                    "bestSellingByQuantity",
-                    t("bill.totalquantity")
-                  )}
-                />
-              </div>,
-              <div className="bills__recap__item inline">
-                <Bar
-                  options={getOptions(
-                    t("bill.bestsellingbyoccurence"),
-                    true,
-                    isMobile(width) ? 1 : 3,
-                    year.value
-                  )}
-                  data={getDataFromObject(
-                    "bestSellingByOccurence",
-                    t("bill.totaloccurence")
-                  )}
-                />
-              </div>,
-              <div className="bills__recap__item inline">
-                <Bar
-                  options={getOptions(
-                    t("bill.bestpatients"),
-                    true,
-                    isMobile(width) ? 1 : 3,
-                    year.value
-                  )}
-                  data={getDataFromObject(
-                    "bestPatientsByPayments",
-                    t("bill.totalpayment")
-                  )}
-                />
-              </div>,
-              <div className="bills__recap__item inline">
-                <Bar
-                  options={getOptions(
-                    t("bill.mostindebtpatients"),
-                    true,
-                    isMobile(width) ? 1 : 3,
-                    year.value
-                  )}
-                  data={getDataFromObject(
-                    "mostIndebtedPatients",
-                    t("bill.totaldebt")
-                  )}
-                />
-              </div>,
-            ]
-          ) : (
-            <div ref={infoBoxRef} className="info-box-container">
-              <InfoBox type="warning" message={t("common.emptydata")} />
-            </div>
-          )}
+          <div className="bills__recap__item inline">
+            {dataByYearIsLoading && <CircularProgress className="loader" />}
+            <Bar
+              options={getOptions(
+                t("bill.bestsellingbyquantity"),
+                true,
+                isMobile(width) ? 1 : 3,
+                year.value
+              )}
+              data={getDataFromObject(
+                "bestSellingByQuantity",
+                t("bill.totalquantity")
+              )}
```

Comment:
I would use a ternary condition here in order to show or a CircularProgress or the Bar component.

Reply:
May I missed your explanation on Jira.

Reply:
But when we replace the chart div totally by the loader it does'nt look nice. That's why I just prefer to show the loader over the bar div. What do you think?

Reply:
Yeah sounds fine and, actually, the result in the UI is good.